flink-core/src/main/java/org/apache/flink/util/FileUtils.java

```diff
@@ -576,6 +577,21 @@ public static Path expandDirectory(Path file, Path targetDirectory) throws IOExc
 		return filterFileVisitor.getFiles();
 	}
 
+	/**
+	 * Absolutize the given path if it is relative.
+	 *
+	 * @param pathToAbsolutize path which is being absolutize if it is an relative path
+	 * @return the absolutized path
+	 */
+	public static Path absolutizePath(Path pathToAbsolutize) throws IOException {
+		FileSystem fs = pathToAbsolutize.getFileSystem();
```

Comment:
nit: this can be put under `if` condition. It is not mandatory.

Reply:
Another option is that the `Path` class itself can provide a utility method like `getAbsolutePath()` because this function depends on its internal component `FileSystem` and `Path#makeQualified` can also make use of it.

Reply:
I thought about that, but I guess it's too late to do API changes (Path is @Public).